utils/src/ooinstall/openshift_ansible.py

```diff
@@ -153,17 +156,17 @@ def write_inventory_vars(base_inventory, multiple_masters, lb):
 def write_proxy_settings(base_inventory):
     try:
         base_inventory.write("openshift_http_proxy={}\n".format(
-                                                            CFG.settings['openshift_http_proxy']))
+            CFG.settings['openshift_http_proxy']))
```

Comment:
I'm not really partial either way, just noticed the general inconsistency with the other changes being made which favored aligning on the parenthesis.